recipe/meta.yaml

```diff
@@ -9,15 +9,14 @@ source:
   # we have to use git urls here to get the submodules needed for the
   # build
   git_url: https://github.com/dmlc/xgboost
-  git_tag: 917b0a7b46954e9be36cbc430a1727bb093234bb
+  git_tag: d6d8be6519185d2e78ba7de5c68e6b7c9733b6f2
   patches:
     # xgboost patches
     - 0001-conda-Unbundle-libxgboost.-dll-dylib-so.patch
     - 0002-Fix-R-package-PKGROOT.patch
```

Comment:
Do you know if it would be possible to upstream these into xgboost as well?

Reply:
It would be nontrivial, since these files affect how XGBoost gets deployed to [CRAN](https://cran.r-project.org/web/packages/xgboost/index.html). To us (XGBoost project maintainers), the priority is to have XGBoost available on CRAN, since vast majority of users of R install packages from CRAN.

Reply:
Meaning `0002-Fix-R-package-PKGROOT.patch`? Do you have thoughts on `0001-conda-Unbundle-libxgboost.-dll-dylib-so.patch`?

Reply:
The first patch can be upstreamed. Let me do that by the next release (1.2.0).

Reply:
Sounds good. Is there an issue for tracking this?

Reply:
I opened https://github.com/conda-forge/xgboost-feedstock/issues/51